build.bat

```diff
@@ -304,8 +304,8 @@ REM USAGE function: output a usage message
 	echo                  path can only be specified before the project/make
 	echo                  files are generated. (This does not run the
 	echo                  installer.^)
-	echo    regadd      : register the driver into the registry unde
-	echo                  'Elasticsearch ODBC' name;
+	echo    regadd      : register the driver into the registry under
+	echo                  'Elasticsearch ODBC' name; use along 'install' param
```

Comment:
nit: "use along" should be something like "use alongside", "use along with" or "use together with"

Same on line 736

Reply:
Thanks, fixed.